src/world/area_iwa/iwa_01/iwa_01_3_splash.c

```diff
@@ -1,21 +1,101 @@
 #include "iwa_01.h"
 
-MAP_RODATA_PAD(1,splash);
-
 #include "world/area_iwa/iwa_01/splash.png.inc.c"
 #include "world/area_iwa/iwa_01/splash.vtx.inc.c"
 #include "world/area_iwa/iwa_01/splash.gfx.inc.c"
 
-API_CALLABLE(N(func_80240398_90FA48));
-void N(gfx_build_splash)(void);
+#define NUM_SPLASHES (20)
+
+typedef struct SplashState {
+    /* 0x00 */ Vec3f pos;
+    /* 0x0C */ f32 scale;
+    /* 0x10 */ Vec3f rot;
+    /* 0x1C */ Vec3f vel;
+} SplashState; // size = 0x28
+
+void N(gfx_build_splashes)(s32 index) {
+    s32 i;
+    s32 alpha;
+    SplashState* splash = (SplashState*)evt_get_variable(NULL, MapVar(15));
```

Comment:
MapVar(15) --> MV_Unk_F defined in enum in the map header (MV_SplashState would also be fine). Check other maps for reference.